docs/source/migration-two-dot.md

```diff
@@ -221,6 +221,42 @@ server.listen().then(({ url }) => {
 });
 ```
 
+<h3 id="existing-schema">Using Schema as Objects</h3>
+
+Some GraphQL server implementations use a GraphQL schema defined as nested objects. Apollo Server 2 accepts these schemas directly to the `ApolloServer` constructor. The following snippet demonstrates how the same schema as above might have been defined and can be passed to Apollo Server.
```

Comment:
My spitballing suggestion here:

> While we recommend the use [schema-definition language (SDL)](https://www.apollographql.com/docs/apollo-server/essentials/schema.html#sdl) for defining a GraphQL schema since we feel it's more human-readable and language-agnostic, Apollo Server 2 also supports schemas which are built with the [`graphql-js`'s `graphql/type` notation](https://graphql.org/graphql-js/type/) by passing a `GraphQLSchema` to the `schema` option of the `ApolloServer` constructor.
>
> For example, using this technique the above schema might be represented and used as: